homedecor_seating/locale/homedecor_seating.it.tr

```diff
@@ -3,5 +3,5 @@ Armchair=Poltrona
 Someone else owns the spot where the other end goes!=
 Someone else owns the spot where the middle or far end goes!=
 Long Sofa=
-Sofa=
 No room to place the sofa!=
+Sofa=
```

Comment:
divano, sofà or canapè

(I just realized the rest hasn't been filled, maybe another PR)

Reply:
I have the following PR already locally done, added there.